src/enemy.hpp

```diff
@@ -13,17 +13,18 @@
 namespace miracle {
 class Enemy {
   public:
-	explicit Enemy(gsl::not_null<le::ServiceLocator const*> services, glm::vec2 target_pos);
+	explicit Enemy(gsl::not_null<le::ServiceLocator const*> services, glm::vec2 target_pos, float move_speed);
 
 	void render(le::Renderer& renderer) const;
-	void move();
+	void move(kvf::Seconds dt);
```

Comment:
Change the name to `tick()` , and don't use the identifier `move` in C++, it might conflict with `std::move()`, despite being in different namespaces. Also looks weird (use `translate()` or something instead).